ParallelRoadTool/UI/UIUtils.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System.Collections.Generic;
```

Comment:
I think we already have some of these things in the UIUtil class, consider merging them to keep only one.

Reply:
Will do. This first version was a quick port from MoveIt just to get it work.